graphql/resolvers/search.ts

```diff
@@ -52,8 +93,20 @@ const resolvers = {
         )
       );
 
+      const termInfo: TermInfo = await prisma.termInfo.findFirst({
+        where: { termId: "" + args.termId },
```

Comment:
There's no need to type-cast here - we know the term ID is a string. 

As a rule of thumb - if you find yourself doing that in Typescript, that probably means your types are inaccurate/not specific enough. We should almost never have to do that

```suggestion
        where: { termId: args.termId },
```